bentoml/_internal/io_descriptors/json.py

```diff
@@ -63,10 +75,59 @@ def default(self, o: _SerializableObj) -> t.Any:
 
 
 class JSON(IODescriptor):
+    """
+    `JSON` defines API specification for the inputs/outputs of a Service, where either inputs will be
+    converted to or outputs will be converted from a JSON representation as specified in your API function signature.
+
+    .. Toy implementation of a sklearn service::
+        # sklearn_svc.py
+        import pandas as pd
+        import numpy as np
+        from bentoml.io import PandasDataFrame, JSON
+        import bentoml.sklearn
+
+        input_spec = PandasDataFrame.from_sample(pd.DataFrame(np.array([[5,4,3,2]])))
+
+        runner = bentoml.sklearn.load_runner("sklearn_model_clf")
+
+        svc = bentoml.Service("iris-classifier", runners=[runner])
+
+        @svc.api(input=input_spec, output=JSON(orient='records'))
+        def predict(input_arr):
+            res = runner.run_batch(input_arr)
+            return {"res":pd.DataFrame(res)}
+
+    Users then can then serve this service with `bentoml serve`::
+        % bentoml serve ./sklearn_svc.py:svc --auto-reload
+
+        (Press CTRL+C to quit)
+        [INFO] Starting BentoML API server in development mode with auto-reload enabled
+        [INFO] Serving BentoML Service "iris-classifier" defined in "sklearn_svc.py"
+        [INFO] API Server running on http://0.0.0.0:5000
+
+    Users can then send a cURL requests like shown in different terminal session::
+        % curl -X POST -H "Content-Type: application/json" --data '[{"0":5,"1":4,"2":3,"3":2}]' http://0.0.0.0:5000/predict
+
+        {"res":"[{\"0\":1}]"}%
+
+    Args:
+        pydantic_model (`pydantic.BaseModel`, `optional`, default to `None`):
+            Pydantic model schema.
+        validate_json (`bool`, `optional`, default to `True`):
+            If True, then use Pydantic model specified above to validate given JSON.
+        json_encoder (`Type[json.JSONEncoder]`, default to `~bentoml._internal.io_descriptor.json.DefaultJsonEncoder`):
+            JSON encoder.
+    Returns:
+        IO Descriptor that in JSON format.
+    """
+
     def __init__(
         self,
         pydantic_model: t.Optional["pydantic.BaseModel"] = None,
         validate_json: bool = True,
+        orient: t.Literal[
```

Comment:
I'd suggest removing this parameter here,  it might be confusing to the user - if user is looking to control the format of a dataframe output, might be better to just use `PandasDataFrame` or convert df to json before returning the function

Reply:
noted